### PR TITLE
Update the End-User Documentation

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by tsv2013 — APPROVED** [review]
Excellent!

File: docs/End-User-Guide.md [hunk 1]
```diff
@@ -1,258 +1,430 @@
...
+
+Alternatively, you can add a page on the design surface. The surface displays the skeleton of a new page at the bottom. To add a new page, you need to [add a question to it](#how-to-add-a-question-to-the-survey).
+
+### How to: Change the question type
+
+Use the element type selector on the question:
+
+<img src="images/survey-creator-element-type-selector.png" alt="Survey Creator - Element type selector" width="50%">
+
+Some question types are non-interchangeable. For example, a [Dropdown](#dropdown) question supports choices, while [Single Input](#single-input) does not. If you switch the question type from Dropdown to Single Input, your choices will be lost. In this case, you can click the **Undo** button to reinstate the previous configuration:
```
Comment (NatalyaSenichkina) [inline docs/End-User-Guide.md:303]:
```suggestion
Some question types are non-interchangeable. For example, a [Dropdown](#dropdown) question supports choices, while [Single Input](#single-input) does not. If you switch the question type from Dropdown to Single Input, your choices are lost. In this case, you can click the **Undo** button to reinstate the previous configuration:
```

**Review by NatalyaSenichkina — COMMENTED** [review]
Good topic. I just simplified a few sentences and made a couple of corrections (to update articles and prepositions).

File: docs/End-User-Guide.md [hunk 2]
```diff
@@ -1,258 +1,430 @@
...
+### Question and Panel Types
 
-![Overview of Test Survey Area](images/survey-editor-test-survey.png)
+The Toolbox contains the question and panel types described below.
 
-If you need to develop the survey in more detail via the JSON file format, you can access an editor via the `JSON Editor` link at the top of the page.
+#### Single Input
 
-![Image of JSON viewer/editor](images/survey-editor-json.png)
+Respondents enter their answer into a single-line text editor. Use the Single Input type for open-ended questions that demand short answers.
```
Comment (albertov05) [inline docs/End-User-Guide.md:67]:
```suggestion
Respondents enter their answers into a single-line text editor. Use the Single Input type for open-ended questions that require short answers.
```

File: docs/End-User-Guide.md [hunk 3]
```diff
@@ -1,258 +1,430 @@
...
+#### Checkbox
 
+Respondents click one or several checkboxes to select answers. Use the Checkbox type for questions that accept multiple answers.
 
-<a id="toolbox"></a>
-## Individual Toolbox Elements
+<img src="images/question-types-checkbox.png" alt="Question types - Checkbox" width="50%">
 
-![Overview of Toolbox](images/survey-editor-toolbox.png)
+#### Radiogroup
```
Comment (albertov05) [inline docs/End-User-Guide.md:82]:
```suggestion
#### Radio Group
```

File: docs/End-User-Guide.md [hunk 4]
```diff
@@ -1,258 +1,430 @@
...
 
-### Single Input
+Respondents use radio buttons to select a single answer. Use the Radiogroup type for questions that can have multiple options but accept only one answer.
```
Comment (albertov05) [inline docs/End-User-Guide.md:84]:
```suggestion
Respondents use radio buttons to select single answers. Use the Radiogroup type for questions that can have multiple options but accept only one answer.
```

File: docs/End-User-Guide.md [hunk 5]
```diff
@@ -1,258 +1,430 @@
...
 
-### Image picker
+#### Ranking 
 
-The survey respondent is presented with a series of images. One of these images can be selected and only the value associated with that image will be recorded in the survey.
-- To provide a numerical output for this element, enter a numeric value for each image in the `Properties` > `choices` > `value` field.
-- Images can be ascribed different values via `Properties` > `Choices` > `Value`
-- Toggle `Properties` > `General` > `Multi select` to allow the survey respondent to choose more than one image
-- Image sizes can be changed via `Properties` > `General` > `Image Height`/`Image Width`
+Respondents drag and drop items from a list to rearrange them according to rank or preference. Use the Ranking type for questions in which respondents must set their order of items.
```
Comment (albertov05) [inline docs/End-User-Guide.md:115]:
```suggestion
Respondents drag and drop items from a list to rearrange them according to rank or preference. Use the Ranking type for questions in which respondents must set the order of items.
```

File: docs/End-User-Guide.md [hunk 6]
```diff
@@ -1,258 +1,430 @@
...
+Expand the **General** category and select **Allow multiple selection**:
 
-### Signature Pad
+    <img src="images/survey-creator-allow-multiple-selection.png" alt="Survey Creator - Allow multiple selection" width="40%">
 
-The Signature Pad survey element allows respondent input to be directly drawn on screen in a pre-set box or area. This is typically used to obtain a signature from the respondent, however it could be used to obtain any hand-drawn input. The size of the box presented to the respondent may be changed by clicking on the `Properties` tab. The color of the pen and background can also be changed. The input from the respondent is recorded in the survey in the following file formats: JPEG, PNG or SVG. This file format may be changed in the `Properties` tab.
+#### Boolean
 
-### Expression
+Respondents switch a Boolean editor to Yes or No. The response is saved in the survey results as `true` for Yes or `false` for No.
```
Comment (albertov05) [inline docs/End-User-Guide.md:137]:
```suggestion
Respondents switch Boolean editors to Yes or No. The response is saved in the survey results as `true` for Yes or `false` for No.
```

File: docs/End-User-Guide.md [hunk 7]
```diff
@@ -1,258 +1,430 @@
...
 
-### Conditional Logic
+<img src="images/question-types-file.png" alt="Question types - File" width="50%">
 
-Every survey element can be triggered by any question within the survey. This allows the `visibility` or `availability` of each element to be conditional on the responses made elsewhere in the survey. For example, `Conditional Logic` would allow for a `Matrix` to appear if a `Boolean` question was answered `Yes`, and an `Image Picker` to appear if the same `Boolean` was answered `No`. Using `Logic` within your survey makes it dynamic and responsive, and enables you to create separate pathways throughout the survey as a whole.
+- **How to: Restrict uploaded file types**          
+The File type is build upon the standard `<input type="file">` HTML element. To restrict file types, the standard element uses the <a href="https://www.w3schools.com/tags/att_input_accept.asp" target="_blank">`accept`</a> attribute. Enter its value into the **Accepted file types** field in the **General** category:
```
Comment (albertov05) [inline docs/End-User-Guide.md:189]:
```suggestion
The File type is built upon the standard `<input type="file">` HTML element. To restrict file types, the standard element uses the <a href="https://www.w3schools.com/tags/att_input_accept.asp" target="_blank">`accept`</a> attribute. Enter its value into the **Accepted file types** field in the **General** category:
```

File: docs/End-User-Guide.md [hunk 8]
```diff
@@ -1,258 +1,430 @@
...
+The Dynamic Matrix type is similar to a [Multiple-Choice Matrix](#multiple-choice-matrix), but respondents can add and remove matrix rows.
 
-![Image of the `Column` > `Edit` properties](images/survey-editor-logic-matrix-column.png)
+<img src="images/question-types-matrix-dynamic-rows.png" alt="Question types - Dynamic Matrix" width="50%">
 
-Once you click `Edit` for a specific column, you will be shown a configuration section for that column:
+#### Multiple Text
 
-![Image of the `Column` > `Edit` properties detail](images/survey-editor-logic-column-detail.png)
+Respondents enter their answers into multiple single-line text editors. Use the Multiple Text type for open-ended questions that demand more than one short answer.
```
Comment (albertov05) [inline docs/End-User-Guide.md:256]:
```suggestion
Respondents enter their answers into multiple single-line text editors. Use the Multiple Text type for open-ended questions that require more than one short answer.
```

File: docs/End-User-Guide.md [hunk 9]
```diff
@@ -1,258 +1,430 @@
...
+Adorners are design-surface controls for survey element manipulation. Use adorners to edit UI texts in place, reorder choices, duplicate and delete elements, specify whether an answer is required, and perform other actions on survey elements. Settings that you specify using adorners are synchronized with settings in the Property Grid, and vice versa.
 
-- [SurveyJS Support Forums](https://surveyjs.answerdesk.io/ticket/list) - useful for searching user questions on specific features
+Each element type has an associated set of adorners. The following image highlights adorners for a [Dropdown](#dropdown) question:
+
+<img src="images/survey-creator-dropdown-adorners.png" alt="Survey Creator - Adorners" width="50%">
+
+### How to: Add a question to the survey
+
+Drag and drop the desired question type from the Toolbox onto the design surface. Alternatively, you can click the **Add Question** button to add a [Single Input](#single-input) question. This button also displays a three-dots icon that allows you to change the type of inserted questions.
```
Comment (albertov05) [inline docs/End-User-Guide.md:285]: What do you mean by "three-dots"? For example, do you mean "ellipsis"?

File: docs/End-User-Guide.md [hunk 10]
```diff
@@ -1,258 +1,430 @@
...
+
+In the Property Grid, select **Survey**, expand the **Pages** category, and click the **Add New** button in the **Pages** editor:
+
+<img src="images/survey-creator-add-new-page.png" alt="Survey Creator - Add new page" width="40%">
+
+Alternatively, you can add a page on the design surface. The surface displays the skeleton of a new page at the bottom. To add a new page, you need to [add a question to it](#how-to-add-a-question-to-the-survey).
+
+### How to: Change the question type
+
+Use the element type selector on the question:
```
Comment (albertov05) [inline docs/End-User-Guide.md:299]: What do you mean by "on the question"? For example, do you mean "in the question"?

File: docs/End-User-Guide.md [hunk 11]
```diff
@@ -1,258 +1,430 @@
...
+
+### How to: Clear the survey
+
+Click the **Clear Survey** button on the Toolbar:
+
+<img src="images/survey-creator-clear-survey.png" alt="Survey Creator - The Clear Survey button" width="50%">
+
+## Preview Tab
+
+The Preview tab allows you to see and take the resulting survey as respondents will do it.
```
Comment (albertov05) [inline docs/End-User-Guide.md:315]: I'd like to simplify this sentence if possible. How about the following?

`The Preview tab allows you to view and take the survey.`

File: docs/End-User-Guide.md [hunk 12]
```diff
@@ -1,258 +1,430 @@
...
+
+### How to: Re-run the preview
+
+After you complete the survey, you can see the survey results in the table or JSON format. Click the Preview Survey Again button above the results to preview the survey again:
+
+<img src="images/survey-creator-preview-survey-again.png" alt="Survey Creator - Preview survey again" width="50%">
+
+### How to: Preview the survey on different devices
+
+Click the **Device** icon and select the desired device from the appeared drop-down menu. You can also click the **Orientation** icon to switch between the landscape and portrait orientations:
```
Comment (albertov05) [inline docs/End-User-Guide.md:327]:
```suggestion
Click the **Device** icon and select the desired device from the appeared drop-down menu. You can also click the **Orientation** icon to switch between the landscape and portrait orientations:
```
```suggestion
Click the **Device** icon and select the desired device from the drop-down menu. You can also click the **Orientation** icon to switch between the landscape and portrait orientations:
```

File: docs/End-User-Guide.md [hunk 13]
```diff
@@ -1,258 +1,430 @@
...
+Makes the selected page editable. If you want to make the page read-only, invert the logic of your condition.
+
+- **Show (hide) question**          
+Makes the selected question visible. If you want to hide the question, invert the logic of your condition.
+
+- **Enable (disable) question**         
+Makes the selected question editable. If you want to make the question read-only, invert the logic of your condition.
+
+- **Make question required**            
+Demands an answer for the selected question.
```
Comment (albertov05) [inline docs/End-User-Guide.md:372]:
```suggestion
Requires an answer for the selected question.
```

File: docs/End-User-Guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (RomanTsukanov) [reply docs/End-User-Guide.md:67]: Single Input displays only one editor and allows respondents to enter only one answer. Suggestions that I left unapplied have similar inaccuracies.

File: docs/End-User-Guide.md (hunk 11 above)
Comment (RomanTsukanov) [reply docs/End-User-Guide.md:315]: It's important that the Preview tab shows the survey as respondents will see it.

File: docs/End-User-Guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (albertov05) [reply docs/End-User-Guide.md:67]: Okay, thank you for the clarification.

File: docs/End-User-Guide.md (hunk 11 above)
Comment (albertov05) [reply docs/End-User-Guide.md:315]: How about the following? (Note that I don't think "will do it" is necessary at the end.)

`The Preview tab allows you to view and take the survey as a respondent.`

**Pull request merged**